pkgs/development/libraries/gstreamer/bad/opencv-4.3.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/ext/opencv/meson.build b/ext/opencv/meson.build
+index f38b55d..05b142e 100644
+--- a/ext/opencv/meson.build
++++ b/ext/opencv/meson.build
+@@ -65,7 +65,7 @@ if opencv_found
+     endif
+   endforeach
+ else
+-  opencv_dep = dependency('opencv4', version : ['>= 4.0.0', '< 4.2.0'], required : false)
++  opencv_dep = dependency('opencv4', version : ['>= 4.0.0', '< 4.4.0'], required : false)
```

Comment:
Can you also send this upstream? https://gitlab.freedesktop.org/gstreamer/gst-plugins-bad/

Reply:
Patch sent to upstream, https://gitlab.freedesktop.org/gstreamer/gst-plugins-bad/-/merge_requests/1235